### PR TITLE
chore(git): extend allowed commit message types

### LINKED ISSUE #9460: Allow `feat` in commit messages
Our commit message policy is based on Angular commit message format, which uses `feat`. As I use default formation in conventional changelog in my plugins, I keep forgetting that Elgg doesn't support it.

### PR DESCRIPTION
**Pull request by hypeJunction**
Adds "feat" as an alias of "feature"
Adds "fixes" and "fixed" as an alias of "fix"

Fixes #9460
Refs #9829

### CODE REVIEW

**Comment by hypeJunction** [comment]
The  docs here https://github.com/Elgg/conventional-changelog say:

> Alias two types to the same section: {feat: 'Features', feature: 'Features'}

I am not sure if we need update anything else there. @ewinslow @mrclay

**Comment by juho-jaakkola** [comment]
LGTM

**Comment by juho-jaakkola** [comment]
Oops, didn't mean to close.

**Comment by mrclay** [comment]
Have you verified this doesn't write multiple "Features" sections in the generated log?

**Comment by mrclay** [comment]
LGTM

File: .scripts/write-changelog.js [hunk 1]
```diff
@@ -5,18 +5,22 @@ var fs = require('fs');
...
+	types: {
+		feat: 'Features',
+		feature: 'Features',
+		perf: 'Performance',
+		docs: 'Documentation',
+		fix: 'Bug Fixes',
+		fixes: 'Bug Fixes',
+		fixed: 'Bug Fixes',
+		deprecate: 'Deprecations',
+		breaks: 'Breaking Changes'
```
Comment (mrclay) [inline .scripts/write-changelog.js]: While at it let's add "performance", "doc", "deprecates", and "break".

**Comment by hypeJunction** [comment]
> Have you verified this doesn't write multiple "Features" sections in the generated log?

I am not 100% sure. The docs say it will, but I don't really know how to test this manually.

**Comment by mrclay** [comment]
LGTM with or without my requested additions.

**Comment by hypeJunction** [comment]
@mrclay added your suggestions

**Pull request merged**